### DISCUSSION TITLE
Incorrect PCB Clearance (on thumb cluster + some walls)

### DISCUSSION THREAD

**Discussion by bosd** [joshreve/dactyl-keyboard#120] [General]
The PCB Clearance is working again after #118
However I notived the cut in the thumb section, is not there.
The outer wall is not cut, to make clearance for the PCB.

datail
![image](https://github.com/joshreve/dactyl-keyboard/assets/11499387/8d9cd88f-a9cc-4f09-a77c-4e038a354dea)

overview:
![image](https://github.com/joshreve/dactyl-keyboard/assets/11499387/ac6306ea-14b1-4b27-979e-2f339e87498b)

A Correct cut of the outer walls, to accomodate the pcb.
Screenshot taken from the main part near controller pcb area.
![image](https://github.com/joshreve/dactyl-keyboard/assets/11499387/b4fa782e-61cd-4e10-a032-596f58155f89)

Might need further investigation,
As inner solids are cut to accomodate the pcb. As well as some outer walls. But is seems not all walls are cut.

**Comment by joshreve** [comment]
Interesting.  The cuts are supposed to be the last step to ensure they get everything.  I'm wondering if it got shifted by another change.
